src/codeu/chat/server/Controller.java

```diff
@@ -56,6 +59,326 @@ public ConversationHeader newConversation(String title, Uuid owner) {
   }
 
   @Override
+  public Integer toggleRemovedBit(Uuid convo, Uuid user){
+    final ConversationHeader foundConvo = model.conversationById().first(convo);
+    final User foundUser = model.userById().first(user);
+
+    Integer result = 0;
+
+    if(foundConvo != null && foundUser != null) {
+      Integer access = foundConvo.accessControls.computeIfAbsent(user, newAccess -> 0);
```

Comment:
Thank you for this, I didn't know `getOrDefault` existed. So is `computeIfAbsent` only for collections?